tfhe/src/keycache/mod.rs

```diff
@@ -38,8 +38,9 @@ pub mod utils {
             named_params_impl!(expose $($const_param),*);
 
             impl NamedParam for $param_type {
-                fn name(&self) -> &'static str {
+                fn name(&self) -> String {
                     named_params_impl!({*self; $param_type} == ( $($const_param),* ));
+                    panic!("Unnamed parameters");
```

Comment:
why this change ? having the panic in the macro which expands to the implementation seemed better ?

Reply:
nevermind, just saw this in another part of the code

we need to have that chat about keycache